lua/cmp_luasnip/init.lua

```diff
@@ -38,8 +38,13 @@ local function get_documentation(snip, data)
 	documentation = table.concat(documentation, "\n")
 
 	doc_cache[data.filetype] = doc_cache[data.filetype] or {}
-	doc_cache[data.filetype][data.snip_id] = documentation
-
+	doc_cache[data.filetype].auto = doc_cache[data.filetype].auto or {}
+	doc_cache[data.filetype].snip = doc_cache[data.filetype].snip or {}
+	if data.auto then
+		doc_cache[data.filetype]['auto'][data.snip_id] = documentation
+	else
+		doc_cache[data.filetype]['snip'][data.snip_id] = documentation
+	end
```

Comment:
Is there an advantage to splitting the docstrings into auto/snip? `snip_id` is unique for all snippets, so theoretically it would be enough to just use that as key

Reply:
Not that I can think of one. I just took it from [at]saadparwaiz1, but since the original PR originated before the use of `snip_id`, I guess that at that time the distinction was needed as the id wasn't unambiguous.
Not quite sure though so let's see if @saadparwaiz1 had another thing in mind.

Reply:
Removed this distinction. Should be fine now, right?

Reply:
Nice, yes that should work :+1: